src/components/Burger/BurgerIngredient/BurgerIngredient.js

```diff
@@ -1,32 +1,33 @@
-import React, { Component } from "react";
-import classes from "./BurgerIngredient.css";
-import PropTypes from "prop-types";
+import React, { Component } from 'react';
+import classes from './BurgerIngredient.css';
+import PropTypes from 'prop-types';
 
-class BurgerIngredient extends Component {
+export class BurgerIngredient extends Component {
   render() {
+    const { type } = this.props;
```

Comment:
Things also helps make the props more explicit in the same way that the destructing does in functional components